api/lambdas/sqs-to-converter/index.js

```diff
@@ -407,3 +409,29 @@ function _log(prefix) {
       ? console.log(`[${prefix}] ${msg}`, ...optionalParams)
       : console.log(`[${prefix}] ${msg}`);
 }
+
+// Keep all capture* functions regardless of usage, so its easier to keep them in sync/the same
+// so later we can move them to a lambda layer
+function captureException(error, captureContext) {
+  const extra = captureContext ? stringifyExtra(captureContext) : {};
+  return Sentry.captureException(error, {
+    ...captureContext,
+    extra,
+  });
+}
+function captureMessage(message, captureContext) {
+  const extra = captureContext ? stringifyExtra(captureContext) : {};
+  return Sentry.captureMessage(message, {
+    ...captureContext,
+    extra,
+  });
+}
+function stringifyExtra(captureContext) {
+  return Object.entries(captureContext.extra ?? {}).reduce(
+    (acc, [key, value]) => ({
+      ...acc,
+      [key]: JSON.stringify(value, null, 2),
+    }),
+    {}
+  );
+}
```

Comment:
hypernit: the `C` programmer in me finds it easier to understand code when it's declared in order of usage (ie I was confused as to why we were no longer using `Sentry.capture*()` until I scrolled to the bottom of the file) - would suggest declaring "utility" funcs at the top of the file

Reply:
LOL, this is a one of those topics... just like naming stuff... 😆 

I tend to put the most important stuff at the beginning of the file - you can see how code after the `handler` is less and less "important".

I find that VSCode and related IDEs make it easier to navigate through functions, so might as well start with the most important piece of code, otherwise one needs to be scanning the whole file to find where to start reading it.

What do you think?

Reply:
Yeah I'm good either way - pros and cons with both.

Ideally GH would have some IDE functionality on PRs, but it's all text based.

Just a preference thing, but having the things that get used later at the beginning of the file is my preference (like consts at the top of a file) - saves time jumping back and forth between function definitions